Render climbing routes

harahu opened the pull request:
Custom rendering for climbing routes. Work in progress, but I thought I'd share my branch as a start.

Closes #11133

harahu committed:
wip

tyrasd commented on modules/renderer/features.js:
```diff
@@ -210,6 +210,10 @@ export function rendererFeatures(context) {
         return tags['piste:type'];
     });
 
+    defineRule('climbing', function isClimbing(tags){
```
I'm hondering: Is a new feature category really warranted for something as rare as the climbing features?

If we decide to add one, we also need to add the respective strings to `core.yaml` (`en -> feature -> climbing -> description / tooltip`, currently around line 950 in the file), so that these texts can be properly displayed and translated.

harahu replied:
You shouldn't read too much intention into the changes I've made in this PR. I'm quite unfamiliar with the iD code base, so my approach to getting rendering of climbing routes to work was just to follow the pattern established by pistes. In other words, this PR is currently just me doing cargo culting.

I want to achieve two things with this PR:
1. get iD to render climbing routes in some pretty way,
2. get iD to understand that climbing routes are routable, such that paths that are only accessible by way of climbing routes aren't identified as routing islands in the editor.

All code that doesn't contribute to those two goals can be stripped out of the PR.

harahu replied:
I dropped the addition of this rule now.

harahu committed:
Update css

harahu committed:
Simplify

harahu committed:
Change red color

tyrasd committed:
minor styling tweak:

make via ferrata markers fully round (like climbing route lines),
use .stroke classed line for the via ferrata markers (like climbing route lines)

tyrasd committed:
Merge branch 'develop' into climbing

tyrasd committed:
add to changelog

tyrasd reviewed (approved):
Looks great, thanks!
